### PR TITLE
Moved to Ghent

### PR DESCRIPTION
**Pull request by rathesDot**
This PR updates all information to match the state when I moved to Ghent on May 1st.

### CODE REVIEW

File: resources/contents/about.md [hunk 1]
```diff
@@ -18,11 +25,10 @@ I started working as a professional web designer and web developer at very young
...
 
 ---
+**Web Developer at [Teamleader](https://www.teamleader.eu/)**\
+*May 2018 - now*
 
-**Web Developer at [netinsiders](https://netinsiders.de)**\
-*Nov 2016 - now*
-
-By the end of 2016, I joined the team of netinsiders as a Web Developer. Now at daytime I work there as a Laravel and Vue Developer and by night I work on my own projects for Aheenam.
+In summer 2018 I started a new chapter and joined Teamleader. This is not only a great opportunity for me to grow as a developer, but also a big new adventure as I am moving to a new city in a new country with a language I still have to learn.
```
Comment (mrnnn) [inline resources/contents/about.md]: Looking forward to meeting you!

**Comment by rathesDot** [comment]
Wooow, you guys are awesome 🎉 Thanks!

File: resources/contents/about.md [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+*Looking for a PDF version? [Click here to download my resume](/resume)*
 
 I'm Rathes Sachchithananthan and I'm a web developer. If you are interested in learning more about me, you will find the information here.
 
-**Name**: Rathes Sachchithananthan, 1992/04/26\
-**from/located**: Haltern am See, Germany / Schleswig, Germany\
-**E-Mail**: [sachchi@rathes.de](mailto:sachchi@rathes.de)\
-**Social Media**: [Facebook]() // [Twitter](https://twitter.com/rswebdesigner) // [Instagram](https://instagram.com/tamizhographer) // [Github](https://github.com/rathesDot)
+**Name**\
+Rathes Sachchithananthan, 1992/04/26\
```
Comment (hannesvdvreken) [inline resources/contents/about.md]: > 04/26

that's... today!

Comment (rathesDot) [reply resources/contents/about.md]: Seems so 😄

**Pull request merged**